src/video-player.js

```diff
@@ -674,6 +676,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
   }
 
   play() {
+    this.playWasCalled = true;
```

Comment:
Is there a better approach then keeping a state of if play was called?
We will have this.playWasCalled=true even if pause/stop where called after play, no?

Reply:
see my first comment